src/shared/components/alert-button/AlertButton.scss

```diff
@@ -0,0 +1,16 @@
+@import 'src/styles/common';
+
+.alertButton {
+  display: inline-block;
+  position: relative;
+  top: 4px;
+  cursor: pointer;
+  padding: 2px;
+  font-size: 0;
+
+  > svg {
+    width: 18px;
+    height: 18px;
```

Comment:
Width and height should be configurable through CSS variables (we can use 18px as a fallback default).